src/SFA.DAS.ApplyService.Application/Apply/Financial/Applications/OpenFinancialApplicationsRequest.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace SFA.DAS.ApplyService.Application.Apply.Financial.Applications
 {
-    public class OpenFinancialApplicationsRequest : IRequest<List<FinancialApplicationSummaryItem>>
+    public class OpenFinancialApplicationsRequest : IRequest<List<Domain.Entities.Apply>>
```

Comment:
I'm going to assume this will evolve into something aside from Apply entity. Reason it was a FinancialApplicationSummaryItem in EPAO was a cut down version for the Dashboard

Reply:
Updated to use the RoatpApplicationSummaryItem in the APR-1269 branch